criteria.md

```diff
@@ -0,0 +1,47 @@
+### Maturity Level 1 (Sandbox)
+ 
+| Category | Criteria Statement |
+|---|---|
+| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
+| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
+| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
+| Access Control | The project's version control system MUST prevent accidental deletion of the default branch. |
+| Build & Release | The project's build and release pipelines MUST NOT execute arbitrary code that is input from outside of the build script. |
```

Comment:
We have this flagged for additional details because it's a valid recurring question. This would not ban semantic executables like ecosystem installers or eval.